srcpkgs/keepassxc/template

```diff
@@ -17,7 +18,8 @@ makedepends="qt5-tools-devel qt5-svg-devel libgcrypt-devel libargon2-devel
  $(vopt_if browser libsodium-devel)
  $(vopt_if keeshare quazip-devel)
  $(vopt_if network libcurl-devel)
```

Comment:
This can probably be dropped anyway, see https://github.com/keepassxreboot/keepassxc/commit/056bbaa92169152ca96466615b1cd77a272998c6

Reply:
Done!